modules/ossm-jaeger-overview.adoc

```diff
@@ -0,0 +1,25 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-jaeger.adoc
+////
+
+[id="ossm-jaeger-overview_{context}"]
+= Jaeger overview
+
+
+Jaeger lets service owners instrument their services to get insights into what their architecture is doing.
+Jaeger is an open source distributed tracing platform that you can use for monitoring, profiling, and troubleshooting modern, cloud-native, microservices-based applications.
+Jaeger is based on the vendor-neutral OpenTracing APIs and instrumentation.
+
+Using Jaeger lets you perform the following functions:
+
+* Monitor distributed transactions
+
+* Perform service dependency analysis
```

Comment:
This won't be provided by Jaeger within OSSM, so may need to drop from the list until we have a separate productised solution with standalone Jaeger.